File: Go/002_Program/main.go
package main

import (
    "fmt"
    "os"
)

func main() {
    fmt.Println("=== Command Line Arguments ===")

    if len(os.Args) < 2 {
        fmt.Println("Usage: program <arg1> <arg2> ...")
        return
    }

    fmt.Printf("Program name: %s\n", os.Args[0])
    fmt.Printf("Arguments: %d\n", len(os.Args)-1)

    for i, arg := range os.Args[1:] {
        fmt.Printf("  Arg %d: %s\n", i+1, arg)
    }
}

File: Rust/002_Program/main.rs
use std::env;

fn main() {
    println!("=== Command Line Arguments ===");

    let args: Vec<String> = env::args().collect();

    if args.len() < 2 {
        println!("Usage: program <arg1> <arg2> ...");
        return;
    }

    println!("Program name: {}", args[0]);
    println!("Arguments: {}", args.len() - 1);

    for (i, arg) in args.iter().skip(1).enumerate() {
        println!("  Arg {}: {}", i + 1, arg);
    }
}
